src/main/resources/templates/categories/dishwashers/dishwashers.ftl

```diff
@@ -1,31 +1,42 @@
 <#include '../../layout.ftl'>
 
 <@common.standardProductForm "Dishwashers">
-  <@govukRadios.radioGroup path="form.applicableLegislation" legendSize="h2">
+
+  <#assign radioGroupClass>
```

Comment:
These bottom margin overrides are to ensure consistent spacing between the last radio button and the 'what kind of label should I create' summary/details underneath it. Radios with no hidden content have no bottom margin, as there's a `last-of-type` CSS rule that removes it, so these overrides ensure visual consistency between different product categories and between energy labels and internet arrows.

Reply:
Is it worth encapsulating that behaviour in a `hasHiddenContent` macro param (or should that be `lastRadioItemHasHiddenContent`?) Then the radioGroup macro can apply the styling override based on that value, rather than the consumer needing to know when to apply what class.
Otherwise just looking at the code without your explanation it's not that clear why this override styling is applied.

Reply:
It's only applicable to radio groups that are above a summary details but yeah some kind of boolean param might make more sense, I'll take a look